src/pages/APIPage.tsx

```diff
@@ -16,7 +16,7 @@ export const APIPage: FunctionComponent = () => {
     const selectedApi = apiConfigurations.find(a => a.displayName === api);
 
     if (!selectedApi) {
-        navigate('/');
+        navigate(process.env.PUBLIC_URL);
```

Comment:
I think this navigate to `/` is fine.
Once we tell react-router where is the base, it will properly prepend anything with the baseurl.